src/renderer/components/dock/logs/controls.tsx

```diff
@@ -77,11 +93,10 @@ const NonInjectedLogControls = observer(({ openSaveFileDialog, model }: Dependen
           onChange={togglePrevious}
           className="show-previous"
         />
-        <Icon
```

Comment:
couldn't keep the icon? "Download" takes up more space where space is at a premium

Reply:
Leaving only 1 icon won't show us that there more than 1 option hidden in the dropdown.

Reply:
I guess the dropdown title(?) can't be an icon, only text?